paparazzi/build.gradle

```diff
@@ -95,6 +95,29 @@ subprojects {
       ])
     }
   }
+
+  // Avoid the native projects in 'libs'
+  plugins.withId('org.jetbrains.kotlin.jvm') {
+    // Create a test task for each supported JDK.
+    [11, 12, 16, 18].each { majorVersion ->
+      def jdkTest = tasks.register("testJdk$majorVersion", Test) {
+        javaLauncher = javaToolchains.launcherFor {
+          languageVersion = JavaLanguageVersion.of(majorVersion)
```

Comment:
Will this also change GradleRunner's JVM?

Reply:
Good question, I'm not sure.  @JakeWharton ?

Reply:
When I was having "fun" with Java and Gradle versions, I added this println to always see what's up:
https://github.com/TWiStErRob/net.twisterrob.gradle/blob/master/test/src/main/kotlin/net/twisterrob/gradle/test/GradleRunnerRule.kt#L159

(I didn't yet do what Jake suggested in his blog in my Gradle setup.)